src/kinetics/ReactionRate.cpp

```diff
@@ -67,7 +78,9 @@ void ArrheniusRate::setParameters(const AnyMap& node, const Units& rate_units)
         Arrhenius::setParameters(AnyValue(), node.units(), rate_units);
         return;
     }
+
     Arrhenius::setParameters(node["rate-constant"], node.units(), rate_units);
+    return;
```

Comment:
The bare `return` at the end of the function can be removed. Also applies to `PlogRate::setParameters`.